méli-mélo/2021-05-steps/steps.css

```diff
@@ -1,168 +1,120 @@
-	margin-top: 20px;
-}
-ol.lst-stps.ld-zr>li:not(:first-child), .lst-stps.ld-zr .lst-stps-itm:not(:first-child) {
-	margin-top: 40px;
+    content: counter(item);
+    counter-increment: item;
 }
 ol.lst-stps>li:before, .lst-stps .lst-stps-itm:before {
-	content: counter(item);
+    content: counter(item);
 }
 ol.lst-stps.ld-zr>li:before, .lst-stps.ld-zr .lst-stps-itm:before {
-	content: counter(item, decimal-leading-zero);
-	counter-increment: li;
-	font-size: 1.6em;
-	margin-left: -2.4em;
-	min-width: 2em;
-}
-ol.lst-stps.ld-zr ol.lst-stps-sub>li:before, .lst-stps.ld-zr .lst-stps-sub .lst-stps-itm:before {
-	margin-left: -3.4em;
-	min-width: 2.8em;
-}
-ol.lst-stps-sub, .lst-stps-sub {
-	clear: both;
-	counter-reset: subitem;
-	margin-top: 15px;
-	padding-left: 0px;
-}
-ol.lst-stps>li:before, ol.lst-stps ol.lst-stps-sub>li:before, .lst-stps .lst-stps-itm:before, .lst-stps-sub .lst-stps-itm:before {
-	border-color: #26374a;
-	border-style: solid;
-	border-width: 3px;
-	float: left;
-	font-family: Lato, sans-serif;
-	font-weight: 600;
-	margin-right: 10px;
-	position: relative;
-	text-align: center;
-}
-ol.lst-stps:not(.ld-zr)>li:before, ol.lst-stps:not(.ld-zr) ol.lst-stps-sub>li:before, .lst-stps:not(.ld-zr) .lst-stps-itm:before, .lst-stps:not(.ld-zr) .lst-stps-sub .lst-stps-itm:before {
-	background-color: #fff;
-	-webkit-border-radius: 50%;
-	-moz-border-radius: 50%;
-	border-radius: 50%;
-	box-sizing: content-box;
-	line-height: 2;
-	margin-left: -3.2em;
-	margin-top: -8px;
-	text-align: center;
-	width: 2em;
+    content: counter(item, decimal-leading-zero);
+    font-size: 1.4em;
+    padding-left: 0.5em;
+}
+ol.lst-stps>li ol.lst-stps-sub, .lst-stps .lst-stps-sub {
+    clear: both;
+    counter-reset: subitem;
+    padding-left: 0px;
+}
+ol.lst-stps>li ol.lst-stps-sub>li:before, .lst-stps .lst-stps-sub .lst-stps-itm:before {
+    counter-increment: subitem;
+    content: counter(item) "" counter(subitem, lower-alpha) "";
+    margin-left: -3em;
+    margin-top: -6px;
+}
+ol.lst-stps:not(.stps-strpd)>li, ol.lst-stps-sub:not(.stps-strpd)>li, .lst-stps:not(.stps-strpd) .lst-stps-itm {
+    margin-top: 20px;
+    min-height: 4em;
+    padding-left: 3.2em;
+    padding-right: 15px;
+}
+ol.lst-stps-sub:not(.stps-strpd)>li, .lst-stps:not(.stps-strpd) .lst-stps-sub .lst-stps-itm {
+    min-height: 3em;
+    padding-left: 2.6em;
+}
+ol.lst-stps>li:before, ol.lst-stps>li ol.lst-stps-sub>li:before, .lst-stps .lst-stps-itm:before {
+    border-style: solid;
+    border-width: 3px;
+    box-sizing: content-box;
+    float: left;
+    font-family: Lato, sans-serif;
+    font-weight: 600;
+    line-height: 2;
+    margin-left: -3.2em;
+    margin-right: 10px;
+    margin-top: -8px;
+    position: relative;
+    text-align: center;
+    width: 2em;
+}
+ol.lst-stps:not(.ld-zr)>li:before, ol.lst-stps:not(.ld-zr)>li ol.lst-stps-sub>li:before, .lst-stps:not(.ld-zr) .lst-stps-itm:before {
```

Comment:
This CSS selector are not matching any element in the demo page:

```css
.lst-stps:not(.ld-zr) .lst-stps-itm:before
```

I think that demo was removed. I can see it was used before